.github/workflows/build_and_test.yml

```diff
@@ -118,6 +118,13 @@ jobs:
       python-version: ${{ matrix.python-version }}
       test-type: ${{ matrix.test-type }}
 
+  test-slurm:
+    needs: [build-linux]
+    uses: ./.github/workflows/test_ert_with_slurm.yml
+    with:
+      os: ubuntu-latest
+      python-version: '3.10'
```

Comment:
it does not work with Python 3.8?

Reply:
It probably does, I was just thinking of saving resources..